packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMainInput.tsx

```diff
@@ -261,6 +269,16 @@ export const TransferPanelMainInput = React.memo(
       }
     }, [sanitizedAmount, value])
 
+    const onSearchPanelClose = useCallback(() => {
+      setTimeout(() => {
+        if (inputFieldRef.current) {
+          inputFieldRef.current.focus()
+        }
+        // the token info takes some time to load
+        // feels awkward if there is no delay
+      }, 500)
+    }, [inputFieldRef])
```

Comment:
Not a big fan of timeout here. If user tab to another element, this will override it. That's bad for accessibility